GOVERNANCE.md

```diff
@@ -0,0 +1,189 @@
+by The Project is released under the Apache 2 open source license,
+developed openly and hosted in public GitHub repositories under the
+[GitHub organization](https://github.com/arviz-devs/arviz). Examples of
+Project Software include the ArviZ code and the Documentation, etc. The Services run by the
+Project consist of public websites and web-services that are hosted
+at [https://arviz-devs.github.io/arviz/](https://arviz-devs.github.io/arviz/) 
+The Project is developed by a team of distributed developers, called
+Contributors. Contributors are individuals who have contributed code,
+documentation, designs or other work to one or more Project repositories.
+Anyone can be a Contributor. Contributors can be affiliated with any legal
+entity or none. Contributors participate in the project by submitting,
+reviewing and discussing GitHub Pull Requests and Issues and participating in
+open and public Project discussions on GitHub, Slack, and Gitter chat rooms. The foundation of Project participation is openness
+and transparency.
+
+There have many Contributors to the Project, whose contributions are listed in the logs of other repositories of the PyMC and projects.
+
+The Project Community consists of all Contributors and Users of the Project.
+Contributors work on behalf of and are responsible to the larger Project
+Community and we strive to keep the barrier between Contributors and Users as
+low as possible.
+
+The Project is formally affiliated with the 501c3 NumFOCUS Foundation
+([http://numfocus.org](http://numfocus.org)),  NumFOCUS is the
+only legal entity that has a formal relationship with the project (see
+Institutional Partners section below).
+
+### Governance
+
+This section describes the governance and leadership model of The Project.
+
+The foundations of Project governance are:
+
+-   Openness & Transparency
+-   Active Contribution
+-   Institutional Neutrality
+
+Traditionally, project leadership was unstructured but primarily driven
+by a subset of Core Contributors whose active and consistent
+contributions have been recognized by their receiving “commit rights” to the
+Project GitHub repositories. In general all Project decisions are made through
+consensus among the Core Developers with input from the Community. 
+
+While this approach has served us well, as the Project grows and faces more
+legal and financial decisions and interacts with other institutions, we see a
+need for a more formal governance model. Moving forward The Project leadership
+will consist of a Random Variables Council. We view this governance model as
+the formalization of what we are already doing, rather than a change in
+direction.
+
+
+Community Architecture
+-----------------------
+
+* N Person steering council (Must be odd, will be decided in first election)
+* Core Contributors 
+* General Contributors
+
+
+Random Variables Council
+------------------------
+The Project will have a Steering Council that consists of Project Contributors
```

Comment:
To be nominated, I think the requirement was to be core dev for a year, which is what is written in eligibility. I think here should be core dev too